blackbaud/skyux-grids
# Adopt `SkyUIConfigService`

Addresses: https://github.com/blackbaud/skyux-grids/issues/30

---

### How to test
- Clone the repo and checkout the branch
- Type `npm i --no-save @blackbaud-internal/skyux-lib-ui-config-service`
- Open *skyuxconfig.json* and set `name` to `"email"` and `auth` to `true`.
- Open *app-extras.module.ts* and import the following:
```
import { SkyUIConfigService } from '@skyux/core';
import { SkyAppUIConfigService } from '@blackbaud-internal/skyux-lib-ui-config-service';
```
- Open *app-extras.module.ts* and add the following provider:
```
{
  provide: SkyUIConfigService,
  useClass: SkyAppUIConfigService
}
```
- Open *grid-visual.component.html* and add the attribute `settingsKey="new-email-designer"` to the `<sky-grid>` component.
- Serve the SPA.
- Change the order of the grid's columns and refresh the page. The order should be preserved.
